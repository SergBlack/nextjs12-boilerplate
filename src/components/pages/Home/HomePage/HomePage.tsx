import type { NextPage } from 'next';

export const HomePage: NextPage = () => {
  return <h1>Hello, World!</h1>;
};
