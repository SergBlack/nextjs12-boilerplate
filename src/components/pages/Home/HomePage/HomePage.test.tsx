import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { HomePage } from './HomePage';

describe('HomePage', () => {
  it('should to match snapshot', () => {
    const { container } = render(<HomePage />);

    expect(container).toMatchSnapshot();
  });
});
