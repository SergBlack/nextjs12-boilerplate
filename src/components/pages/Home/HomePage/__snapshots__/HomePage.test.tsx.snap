// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage should to match snapshot 1`] = `
<div>
  <h1>
    Hello, World!
  </h1>
</div>
`;
